crates/sc-consensus-subspace/src/verifier.rs

```diff
@@ -33,6 +34,10 @@ use subspace_core_primitives::crypto::kzg::Kzg;
 use subspace_core_primitives::{BlockNumber, PublicKey, RewardSignature};
 use subspace_proof_of_space::Table;
 use subspace_verification::{check_reward_signature, verify_solution, VerifySolutionParams};
+use tokio::sync::Semaphore;
+
+/// This corresponds to default value of `--max-runtime-instances` in Substrate
+const BLOCKS_LIST_CHECK_CONCURRENCY: usize = 8;
```

Comment:
maybe `MAX_BLOCKS_VERIFICATIONS`?

Reply:
Not really, the verification is running at higher concurrency, this is just for block list check that requires runtime API calls and is bound by number of runtime instances.